.gitignore

```diff
@@ -1,4 +1,6 @@
-# Godot 4+ specific ignores
+# Godot auto generated files
+*.gen.*
+.import/
```

Comment:
`.import/` is left over from Godot 3, we don't need that one, and it could be removed from godot-cpp as well